packages/mui-material/src/AvatarGroup/AvatarGroup.js

```diff
@@ -105,7 +105,7 @@ const AvatarGroup = React.forwardRef(function AvatarGroup(inProps, ref) {
   const extraAvatars = Math.max(totalAvatars - clampedMax, totalAvatars - maxAvatars, 0);
   const extraAvatarsElement = renderSurplus ? renderSurplus(extraAvatars) : `+${extraAvatars}`;
 
-  const additionalAvatarSlotProps = slotProps.additionalAvatar ?? componentsProps.additionalAvatar;
+  const additionalAvatarSlotProps = slotProps.additionalAvatar ?? componentsProps?.additionalAvatar;
```

Comment:
Should we implement the `additionalAvatar` slot properly? I think `surplus` is a better name. This would mean:

- Providing `slots.surplus` and `slotProps.surplus`
- Using `useSlot` hook
- Adding tests for the slot

What do you think @lhilgert9?

@mnajdova tagging you as the owner of the Avatar component.

Reply:
Sounds logical to me and would also ensure uniformity for all components and offer the end user more customization options.👍🏼

Reply:
Let's implement it 🙌🏼

Reply:
@DiegoAndai I have added `surplus` as `slots` and `slotProps` prop. I have marked `slotProps.additionalAvatar` as `@deprecated`. I also modified the codemod to change `additionalAvatar` to `surplus`. I just didn't know which tests you wanted to implement. If you tell me which ones we need there, I'll implement them too.